packages/gasket-cli/src/index.d.ts

```diff
@@ -196,7 +195,7 @@ declare module '@gasket/engine' {
     prompt(
       context: CreateContext,
       utils: {
-        prompt: Inquirer,
+        prompt: (prompts: Array<Record<string, any>>) => Promise<Record<string,any>>,
```

Comment:
This doesn't seem like a very helpful type. Can we find a better way to import the type instead?

Reply:
Or add `@types/inquirer` as a dependency?

Reply:
Looks like `@types/inquirer` has a `PromptFunction` type that maybe is what this is?

Reply:
We set up the prompt function conditionally [here](https://github.com/SBoudrias/Inquirer.js/#inquirercreatepromptmodule---prompt-function). We are using inquirer@6 in the CLI currently, which does not export types. There is a `@types/inquirer`, but then it becomes a necessary dependency for users to add, unless we make it a direct dependency of the CLI, which doesn't feel right either.

Reply:
The `PromptFunction` type is in v9 of @types/inquirer [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/inquirer/index.d.ts#L53), but it doesn't exist in v6 version [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/cfde12269ccf42b9c1db2a04e260a4f8ed4ccf62/types/inquirer/index.d.ts). Maybe I could use `PromptModule` from v6? But, either way, this will require a hard dependency of the cli on this @types/inquirer to ensure it is synced up.

Reply:
We will investigate further when upgrading
https://github.com/godaddy/gasket/issues/618